packages/trpc/server/routers/viewer/organizations/create.handler.ts

```diff
@@ -21,7 +21,7 @@ type CreateOptions = {
   input: TCreateInputSchema;
 };
 
-const vercelCreateDomain = async (domain: string) => {
+const vercelCreateDomain = async (domain: string): Promise<boolean> => {
```

Comment:
```suggestion
const vercelCreateDomain = async (domain: string) => {
```
Lets just use the inferred type

Reply:
I made this change because I was getting warnings from the linter. But yes, as I have looked at many other places in the code, inferred type has been used. So I will make this change.